internal/services/ingest.go

```diff
@@ -389,33 +397,55 @@ func (m *ingestService) ingestProcessedData(ctx context.Context, ledgerIndexer *
 		opByID := make(map[int64]types.Operation)
 		stellarAddressesByOpID := make(map[int64]set.Set[string])
 
+		stateChanges := make([]types.StateChange, 0)
+
 		// 1. Build the data structures needed for the DB insertions
 		for participant := range indexerBuffer.Participants.Iter() {
-			if !indexerBuffer.Participants.Contains(participant) {
-				continue
-			}
```

Comment:
If we process everything in the indexer, we can re-add this filter here to decrease the data set being sent to the SQL queries.

Reply:
I think the IF condition should be a check with our future bloom filter/in-memory account cache